src/generators.cpp

```diff
@@ -45,8 +46,14 @@ void OnCudaError(cudaError_t error) { assert(false); }
 
 static bool _ = (Ort::InitApi(), false);
 
+static OrtLoggingLevel GetDefaultOrtLoggingLevel() {
+  bool ort_verbose_logging = false;
+  GetEnvironmentVariable("ORTGENAI_ORT_VERBOSE_LOGGING", ort_verbose_logging);
```

Comment:
Can the existing ORT environment variable names be used or do the environment variable names have to be ORT GenAI specific?

Reply:
I'm not aware of an existing general ORT environment variable that controls the log level. if there is one, maybe we don't need this at all.